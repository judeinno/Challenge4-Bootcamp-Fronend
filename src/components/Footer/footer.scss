.footer {
    margin-top: 120px;
    font-family: "Roboto Condensed", sans-serif;
}