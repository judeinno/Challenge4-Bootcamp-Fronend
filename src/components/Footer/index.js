import React from 'react';
import './footer.scss';

const Footer = () => (
  <div className="card text-center footer">
    <div className="card-body">
      <h5 className="card-title">StackOverFlow-lite, Ask anything!</h5>
      <p className="card-text">2018 StackOverFlow-lite</p>
      <a href="/" className="btn btn-primary">About Us</a>
    </div>
  </div>
);


export default Footer;
