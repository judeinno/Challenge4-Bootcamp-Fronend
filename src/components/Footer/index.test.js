import React from 'react';
import { shallow } from 'enzyme';
import Footer from './index';

it('testing Footer', () => {
  const FooterComponent = shallow(
    <Footer />,
  );
  expect(FooterComponent).toMatchSnapshot();
});
