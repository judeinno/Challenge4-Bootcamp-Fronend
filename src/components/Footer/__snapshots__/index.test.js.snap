// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Footer 1`] = `
<div
  className="card text-center footer"
>
  <div
    className="card-body"
  >
    <h5
      className="card-title"
    >
      StackOverFlow-lite, Ask anything!
    </h5>
    <p
      className="card-text"
    >
      2018 StackOverFlow-lite
    </p>
    <a
      className="btn btn-primary"
      href="/"
    >
      About Us
    </a>
  </div>
</div>
`;
